### DISCUSSION TITLE
mevo plus rois?

### DISCUSSION THREAD

**Discussion by Wrightn13** [springbok/MLM2PRO-GSPro-Connector#95] [Q&A]
is there a way to add more boxes in the roi? id like to see the club path and face to path that i get from the mevo plus but its not an option to put another box. i would like to include things such as club path club face to path and things like that

**Comment by mickilus** [comment]
@springbok I would also love if there were some more ROIs transferred over to GSPro. Club Path, Face to Path, Face to Target specifically would be  awesome!

**Comment by cubev** [comment]
+1 
& and thx a lot for your great work

**Comment by wonder99** [comment]
Face to path is not supported by GSPRO, but adding face to target, club path, and angle of attack should all be possible.  I'll have a look.
Let me know if you want all 3

ref: https://gsprogolf.com/GSProConnectV1.html

**Comment by wonder99** [comment]
please send me a high resolution screenshot of the window with all the metrics.

**Reply by Wrightn13** [reply]
Hi I will get you the screenshots tomorrow for the additional information. Thank you for taking the time.

**Reply by Wrightn13** [reply]
![Screenshot 2024-09-05 201028](https://github.com/user-attachments/assets/59774c7b-4f7d-4808-9220-c9de68d14325)
here is a screenshot of the Flightscope page let me know if you might need more info

**Reply by wonder99** [reply]
Thanks. So cross-referencing that with the GSPRO API, I can send FaceToTarget, ClubPath, and AoA.  We could send FaceToPath by hijacking a different metric name, such as HorizontalFaceImpact.  It would look a little weird in GSPRO, but it would be there.

**Comment by gsgorveatt1234** [comment]
Wondering if anyone knows how to make a traineddata file for this image. Can't seem to get roi's to work with the preset files. Please help if possible. 
![mevo plus](https://github.com/user-attachments/assets/770a73a8-818a-46cd-af8b-e68673aa34de)

**Reply by wonder99** [reply]
Yes but I need more samples. There is no green e nor a 6. Send me a few more

**Reply by gsgorveatt1234** [reply]
Is this more of what our looking for. Appreciate the help.
![1](https://github.com/user-attachments/assets/556226c7-a3df-40b7-8dc2-73e08861f2d2)

![2](https://github.com/user-attachments/assets/fa679b2e-ac75-4185-a91c-2a10393e5610)

**Reply by wonder99** [reply]
Perfect, will do

**Reply by gsgorveatt1234** [reply]
Thank you. Really appreciate this.

**Reply by gsgorveatt1234** [reply]
I tried creating the data file myself last night by following a video you posted in forum. Had no luck with it though. Hopefully you have better luck then me.

**Reply by wonder99** [reply]
Ya, I just tried to train it, but the images are too small--the characters are only about 8x12 pixels, so unfortunately, we can't do it.

**Reply by gsgorveatt1234** [reply]
Ok. Thanks for trying.

**Reply by gsgorveatt1234** [reply]
Is there anyway of combining this connector with my connector. They both run on openapi. I was curious about the relay server option you have. Would this work. Just need a way to connect and disconnect from gspro when mode changes from full swing to putting. Want to use you web cam putting option
